tests/cstor/snapshot/suite_test.go

```diff
@@ -40,7 +42,7 @@ var (
 	scName                = "test-cstor-snap-sc"
 	openebsCASConfigValue = `
 - name: ReplicaCount
-  value: 1
+  value: count
 - name: StoragePoolClaim
   value: test-cstor-snap-sparse-pool
```

Comment:
remember any cluster scoped resource needs to have its name generated. This will avoid clashes when tests are run in parallel.

Reply:
i will do this as part of next PR.